CourseTemplate/CourseTemplate.Data/CorseTemplateRepository.cs

```diff
@@ -8,12 +8,7 @@ public class CourseTemplateRepository : ICourseTemplateRepository
 
 	public CourseTemplateRepository(CourseTemplateDbContext context)
 	{
-		_context = context ?? throw new ArgumentNullException(nameof(context));
-	}
-
-	public CourseTemplateRepository()
-	{
-		_context = new CourseTemplateDbContext();
```

Comment:
How this impact on migrations applying?

Reply:
This should be deleted in order to launch correctly application in docker. All configuration are done in startup